randovania/data/json_data/prime3.txt

```diff
@@ -3631,7 +4483,7 @@ Asset id: 11846340785047609812
       Trivial
```

Comment:
For the two Transit Stations above:

- 1-A
  - Add node "Tram to Pirate Research - Transit Station 1-B" and connect to door as Trivial.
  - Connect door to tram as trivial
- 1-B
  - Add node "Tram to Pirate Research - Transit Station 1-A" and connect to door as Trivial.
  - Connect door to tram as trivial